website/docs/how-to/how-to-add-sso-saml.md

```diff
@@ -125,16 +146,63 @@ You need to fill out the following options:
 - Single Logout Url: https://**[region]**.app.unleash-hosted.com/**[instanceName]**/auth/saml/logout/done
 - SP Issuer: https://**[region]**.app.unleash-hosted.com/**[instanceName]**
 
-Next upload the public Certificate you generated in the previous step (`cert.pem`) and save the Okta SAML settings. Upon completion of this step you should be provided with the ability to view setup instructions and now you should be provided with a "Identity Provider Single Logout URL"
+Next upload the public Certificate you generated in the previous step (`cert.pem`) and save the Okta SAML settings. Upon completion of this step you should be
+provided with the ability to view setup instructions and now you should be provided with a "Identity Provider Single Logout URL"
 
-![SAML 2.0 Okta sing-out url](/img/sso-saml-okta-signout-url.png)
+![SAML 2.0 Okta sign-out url](/img/sso-saml-okta-signout-url.png)
 
 ### Step 3: Configure Single-Sign-Out in Unleash
 
 Go back to Unleash Admin Dashboard and navigate to `Admin Menu -> Single-Sign-On -> SAML`. Fill in the values captured in the "Single Logout URL" from Okta.
 
-In the "Service Provide X.509 Certificate" field you should insert the value of your private key (`private-pem`). This is required in order to make Unleash able to sign logout requests.
+In the "Service Provide X.509 Certificate" field you should insert the value of your private key (`private-pem`). This is required in order to make Unleash able
+to sign logout requests.
 
-![SAML 2.0 Okta sing-out config](/img/sso-saml-okta-signout-unleash.png)
+![SAML 2.0 Okta sign-out config](/img/sso-saml-okta-signout-unleash.png)
 
-After you save these settings users will now be redirected to your IdP (Okta) and back to Unleash again after successfully signing out.
+After you save these settings users will now be redirected to your IdP (Okta) and back to Unleash again after
+successfully signing out.
+
+## Configuration via Environment variables (Since Unleash Enterprise 6.1.0)
```

Comment:
Should we just add a link to the other page? I usually try to avoid having the same doc in two places as one of them tends to drift apart